src/main.ts

```
@@ -0,0 +1,26 @@
+    </div>
+    <h1 class="text-4xl font-bold text-gray-800 mb-8">Vite + TypeScript</h1>
+    <div class="card bg-white p-6 rounded-lg shadow-md mb-8 w-full max-w-md">
+      <button id="counter" type="button" class="w-full py-2 px-4 bg-blue-500 text-white font-medium rounded-md hover:bg-blue-600 transition-colors focus:outline-none focus:ring-2 focus:ring-blue-300"></button>
+    </div>
+    <p class="text-sm text-gray-500 italic">
+      Click on the Vite and TypeScript logos to learn more
+    </p>
+  </div>
+`;
+
+setupCounter(document.querySelector<HTMLButtonElement>("#counter")!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)